lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -320,6 +450,19 @@ def get_properties(autoscaling_group):
     properties['healthcheck_type'] = autoscaling_group.get('HealthCheckType')
     properties['default_cooldown'] = autoscaling_group.get('DefaultCooldown')
     properties['termination_policies'] = autoscaling_group.get('TerminationPolicies')
+    properties['target_group_arns'] = autoscaling_group.get('TargetGroupARNs')
+    if properties['target_group_arns']:
+        region, ec2_url, aws_connect_params = get_aws_connection_info(module, boto3=True)
+        elbv2_connection = boto3_conn(module,
+                                      conn_type='client',
+                                      resource='elbv2',
+                                      region=region,
+                                      endpoint=ec2_url,
+                                      **aws_connect_params)
+        target_groups = elbv2_connection.describe_target_groups(TargetGroupArns=properties['target_group_arns'])['TargetGroups']
```

Comment:
Are AWSRetry and pagination both required? We have no current examples of that, although guess it would be 
```
AWSRetry(tries=5, delay=5, backoff=2.0))(tg_paginator.paginate)(TargetGroupArns=asg['target_group_arns'])
```